docs/software-development-lifecycle.mdx

```diff
@@ -3,4 +3,40 @@ id: software-development-lifecycle
 title: Software Development Lifecycle
 ---
 
-TODO
+In general, a developer should a WIP (work in progress) limit of 1 story at a time (In Development through Complete). The developer claims ownership of that story through the entire development cycle.
+
+1. Backlog
+    * This list holds stories for future work
+    * All new stories should be added here
+    * These stories require additional planning or are too far out that they are not ready to be worked on
+1. Ready For Development
+    * These are stories that are ready to be worked on
+    * All stories in this state should be pointed
+    * Storiees in this state will be scheduled to be worked on during a current or future interation
+    * When a developer is ready for a new story they will take one from this list that is assigned to them and for the current interation
+    * If there are no stories assigned to the developer they can pick an unassigned story
+1. In Development
+    * These are stories that are actively being worked on by a developer
+1. QA (Quality Assurance)
+    * Once a developer has finished work on a story it is moved to the QA Ready column
+    * The developer opens a Pull Request from the feature branch into develop
+    * When testing, QA moves the card to the QA Testing column and will manually test the story and create automated tests
```

Comment:
Typically, I create the automated tests after the story is Accepted. I do create the test cases in this step. Possibly update it to the following:
* When testing, QA moves the card to the QA Testing column and will manually test the story and create the test cases that will define the Automation work after the card is moved to Accepted. (or something like that)